data/windows.yaml

```diff
@@ -195,6 +195,43 @@ urls:
 - 'https://technet.microsoft.com/en-us/library/cc786702(WS.10).aspx'
 - 'http://gladiator-antivirus.com/forum/index.php?showtopic=24610'
 ---
+name: WindowsCIMRepositoryFiles
+doc: |
+  Windows Common Information Model (CIM) repository.
+
+  Persistent database that holds the schema, also called the object repository or class store,
+  that models the managed environment and defines every piece of data exposed by WMI.
+
+  This definition does not specify the copies of the CIM repository that are stored in system restore points.
+sources:
+- type: FILE
+  attributes:
+    paths:
+      # Windows 95 OSR 2.5, 98, Millennium Edition (Me)
+      - '%%environ_windir%%\System\Wbem\Repository\cim.rep'
+      # Windows NT4 and 2000
+      - '%%environ_systemroot%%\System32\wbem\Repository\CIM.REC'
+      - '%%environ_systemroot%%\System32\wbem\Repository\CIM.REP'
+      # Windows Vista and later
+      - '%%environ_systemroot%%\System32\wbem\Repository\INDEX.BTR'
+      - '%%environ_systemroot%%\System32\wbem\Repository\INDEX.MAP'
+      - '%%environ_systemroot%%\System32\wbem\Repository\MAPPING.VER'
+      - '%%environ_systemroot%%\System32\wbem\Repository\MAPPING[1-3].MAP'
```

Comment:
Does plaso understand this syntax?

Reply:
I assume with "this syntax" you mean the "[1-3]" notation?

It "should" support it.

FYI plaso uses the dfvfs.FileSystemSearcher:

* https://github.com/log2timeline/plaso/blob/b7bf7b7f618a649387ad50404e7db7709540736a/plaso/engine/artifact_filters.py#L95
* https://github.com/log2timeline/plaso/blob/b7bf7b7f618a649387ad50404e7db7709540736a/plaso/engine/extractors.py#L602

That uses:

* https://github.com/log2timeline/dfvfs/blob/master/dfvfs/lib/glob2regex.py